test/cache_test.go

```diff
@@ -3,14 +3,12 @@ package test
 import (
 	"testing"
 
-	"github.com/coredns/coredns/plugin/test"
-
 	"github.com/miekg/dns"
 )
 
 func TestLookupCache(t *testing.T) {
 	// Start auth. CoreDNS holding the auth zone.
-	name, rm, err := test.TempFile(".", exampleOrg)
```

Comment:
this seems a good change, but can you split it off from this PR?

(also there are 2 `TempFile`'s, we should remove one)